### PR TITLE
feat(core): Enable new artifacts workflow in bakeManifest

### PR DESCRIPTION
**Pull request by Jammy-Louie**

### CODE REVIEW

**Comment by Jammy-Louie** [comment]
There is corresponding ORCA PR

**Comment by Jammy-Louie** [comment]
Screen Shot 2019-06-21 at 3 31 09 PM

File: app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+interface IBakeManifestStageFormProps {
+  updatePipeline: (pipeline: IPipeline) => void;
+}
+
+export class BakeManifestStageForm extends React.Component<
+  IBakeManifestStageFormProps & IFormikStageConfigInjectedProps
+> {
+  public readonly templateRenderers = ['HELM2'];
+
+  public constructor(props: IStageConfigProps & IFormikStageConfigInjectedProps) {
```
Comment (maggieneterval) [inline app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx]: can probably get rid of this constructor

File: app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        {
+          account: '',
+          id: '',
+        },
+      ]);
+    }
+  }
+
+  private onTemplateArtifactEdited = (artifact: IArtifact, index: number) => {
+    this.props.formik.setFieldValue('inputArtifacts[' + index + '].id', null);
```
Comment (maggieneterval) [inline app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx]:
```suggestion
    this.props.formik.setFieldValue(`inputArtifacts[${index}].id`, null);
```
*Possibly* more readable but :woman_shrugging:

File: app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx (hunk 1 above)
Comment (Jammy-Louie) [reply app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx]: 👍

File: app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx (hunk 2 above)
Comment (Jammy-Louie) [reply app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/BakeManifestStageForm.tsx]: 👍

**Pull request merged**